### DISCUSSION TITLE
Discord Whitelist check.

### DISCUSSION THREAD

**Discussion by safestring** [gameserverapp/Platform#178] [Suggestions]
Basically a really useful feature for us would be to have it so to join the ark server they have to be both apporved whitlisted and in the discord. I dont even think this is possible but would help.

**Comment by mvaessen** [comment]
Discord is currently rolling out a big privacy update, restricting the options to "see" whether people are in a Discord server or not.
While it's technically possible, I doubt Discord would allow such practises. Nevertheless, this is something we can look into.
